frontend/src/global_styles/content/_buttons.sass

```diff
@@ -90,7 +90,7 @@ a.button
   &.-alt-highlight
     @include button-style(var(--button--alt-highlight-background-color), var(--button--alt-highlight-background-hover-color), var(--button--alt-highlight-font-color))
     @include button-style(var(--button--alt-highlight-background-color), var(--button--alt-highlight-background-hover-color), var(--button--alt-highlight-font-color))
-    border-color: var(--button--alt-highlight-border-color)
+    border-color: var(--button--alt-highlight-background-color)
```

Comment:
Are you sure that this is accepted by the product team? Please double check with them in doubt.